tasks/main.yml

```diff
@@ -8,8 +8,9 @@
 - name: Expand filesystem to fill disk
   command: raspi-config --expand-rootfs
   when: unpartitioned.stdout != "0.00GB"
-  notify:
-  - Restart
-  - Wait for restart to finish
+
+- name: Reboot after file system change
+  reboot:
+  when: unpartitioned.stdout != "0.00GB"
 
 - meta: flush_handlers
```

Comment:
Does this not do what https://docs.ansible.com/ansible/latest/modules/meta_module.html says it should do? My read of that documentation is that this `meta` handler should trigger the reboot if needed.

Regardless, I think `reboot` is clearer, so could you please remove this line and update the PR?

Reply:
Reading the documentation I agree with you, the handler should have been called immediately. I really don't know why it wasn't but it was reproducable. I was using ansible v2.7.6.

Anyway, I've removed the line to flush the handlers.